### PR TITLE
config: Internet is danger, keep safe and stay with friends

### PR DESCRIPTION
**Pull request by Xuanwo**
Signed-off-by: Xuanwo <github@xuanwo.io>

I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

It's dangerous to expose `0.0.0.0` by default, this PR will replace all `0.0.0.0` in code to `127.0.0.1` to keep safe.

## Changelog

- Improvement

## Test Plan

Unit Tests

Stateless Tests

### CODE REVIEW

File: scripts/ci/wait_tcp.py [hunk 1]
```diff
@@ -12,7 +12,7 @@ def tcp_ping(port, timeout):
     while time.time() - now < timeout:
         try:
             with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as sock:
-                sock.connect(('0.0.0.0', port))
+                sock.connect(('127.0.0.1', port))
```
Comment (drmingdrmer) [inline scripts/ci/wait_tcp.py:15]: This is a client-side tcp for testing purpose only. Leaving it 0.0.0.0 may be better?

Comment (Xuanwo) [inline scripts/ci/wait_tcp.py:15]:
```suggestion
                sock.connect(('0.0.0.0', port))
```

**Review by drmingdrmer — APPROVED** [review]
Great again!

**Comment by Xuanwo** [comment]
@mergify update

**Comment by BohuTANG** [comment]
Mergify bot can not save us on conflicting files :laughing:

**Review by drmingdrmer — APPROVED** [review]

**Comment by BohuTANG** [comment]
Some tests failed:
```
---- configs::test_load_config stdout ----
thread 'configs::test_load_config' panicked at 'assertion failed: `(left == right)`
  left: `"fv-az190-599"`,
 right: `"127.0.0.1"`', metasrv/tests/it/configs.rs:87:5
stack backtrace:
```

**Review by lichuang — APPROVED** [review]

**Comment by BohuTANG** [comment]
Oops, unit test not passed

**Pull request merged**